docs/testing/stencil-testrunner/05-e2e-testing.md

```diff
@@ -85,19 +85,11 @@ Use the "piercing" selector `>>>` to query for an object inside a component's sh
 const el = await page.find('foo-component >>> .close-button');
 ```
 
-:::caution
-Make sure to only use `>>>` once in your query. Puppeteer does not support nested deep selectors. Instead, separate the calls into two queries, e.g.:
+By default Stencil will look into all shadow roots to find the element. However if you want to restrict your query by specifying the shadow root of a particular component, you can chain multiple deep selectors within the same query, e.g.:
```

Comment:
Will this change only be in affect for Stencil v4.14 and up? If so, shouldn't we leave the other docs as-is?

Reply:
yes 🙈